Examples/tvOS Integration/Podfile.lock

```diff
@@ -1,15 +1,15 @@
 PODS:
-  - OneMobileSDK-Swift3 (2.6.1375):
-    - VideoRenderer (= 1.1)
-  - VideoRenderer (1.1)
+  - OneMobileSDK-Swift3 (2.16.1780):
```

Comment:
Beta have to be here.
I don't see part that reflects that `isFinished` is fixed.

Reply:
@AndriiMoskvin 👍